interactions/client.py

```diff
@@ -555,15 +555,30 @@ async def autocomplete_choice_list(ctx, user_input: str = ""):
 
         :param name: The name of the option to autocomplete.
         :type name: str
-        :param command: The command or commnd ID with the option.
-        :type command: Union[ApplicationCommand, int]
+        :param command: The command, command ID, or command name with the option.
+        :type command: Union[ApplicationCommand, int, str]
         :return: A callable response.
         :rtype: Callable[..., Any]
         """
         _command: Union[Snowflake, int] = (
             command.id if isinstance(command, ApplicationCommand) else command
         )
 
+        if isinstance(command, ApplicationCommand):
+            _command: Union[Snowflake, int] = command.id
+        elif isinstance(command, str):
+            _command_obj = self.http.cache.interactions.get(command)
+            if not _command_obj:
+                _sync_task = ensure_future(self.synchronize(), loop=self.loop)
+                while not _sync_task.done():
+                    pass  # wait for sync to finish
+                _command_obj = self.http.cache.interactions.get(command)
+                if not _command_obj:
+                    raise InteractionException(6, message="The command does not exist")
+            _command: Union[Snowflake, int] = int(_command_obj.id)
+        else:
+            _command: Union[Snowflake, int] = command
+
```

Comment:
```suggestion
        elif isinstance(command, int):
            _command: Union[Snowflake, int] = command
        else:
              raise ValueError("You can only insert string, intergers and ApplicationCommands here!") # TODO: move to custom error formatter
```

Reply:
I'd recommend checking if it is actually an integer and if not raising an exception.